AltBeacon.Library/src/AltBeacon/Beacon/BeaconParser.cs

```diff
@@ -416,8 +417,7 @@ public BeaconParser SetBeaconLayout(string beaconLayout)
 
                 if (!found)
                 {
-                    ////TODO LogManager
-                    ////LogManager.d(Tag, "cannot parse term " + term);
+                    Logger.Debug("cannot parse term " + term);
```

Comment:
I don't think you have to "add LogManager.VerboseLoggingEnabled condition to every logger".  It is only really necessary inside the parser where performance counts.